jbpm-wb-human-tasks/jbpm-wb-human-tasks-backend/src/main/java/org/jbpm/workbench/ht/backend/server/RemoteTaskServiceImpl.java

```diff
@@ -77,6 +80,33 @@ public TaskSummary getTask(String serverTemplateId,
         }
     }
 
+    @Override
+    public TaskSummary getTaskWithSLA(String serverTemplateId, String containerId, Long taskId, Long workItemId) {
```

Comment:
I wonder if we should enhance _getTask_ instead, to make the data consistent in all calls. wdyt?